consumers/src/main/java/uk/gov/moj/sdt/cmc/consumers/client/impl/JudgementRequestServiceService.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.moj.sdt.cmc.consumers.client.impl;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.moj.sdt.cmc.consumers.api.CMCApi;
+import uk.gov.moj.sdt.cmc.consumers.api.IJudgementService;
+import uk.gov.moj.sdt.cmc.consumers.request.judgement.JudgementRequest;
+import uk.gov.moj.sdt.cmc.consumers.response.judgement.JudgementResponse;
+
+@Service("JudgementRequestService")
+public class JudgementRequestServiceService implements IJudgementService {
```

Comment:
Should this be named `JudgementRequestService` rather than `JudgementRequestServiceService`?

Reply:
done